### PR TITLE
App V0.5.0

### PR DESCRIPTION
**Pull request by dansjack**
## Summary

Version 0.5.0 of the app adds the following:
* Navigation component
* Compare locations fragment
* Updated about page
* Fixed issue where active cases were pulled at state level, but overall cases were pulled at county and country level (now all three use overall cases)
* Adds a lot more tests
* Adds more javadocs
* Adds location icon in nav bar

## Checklist
- [x] All tests passing locally
- [x] All tests passing on CircleCI
- [x] Assigned at least 2 reviewers

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (unicornsleuth) [inline .idea/misc.xml:3]: is there a reason for changing this?

**Review by unicornsleuth — DISMISSED** [review]
Everything looks good! Question about the .idea/misc.xml, but it *probably* isn't important

**Review by AllisonBC — DISMISSED** [review]
We should make a group decision on how we want to handle the idea directory files, but I don't think that has to preclude this merge. Thanks Dan!

File: .idea/misc.xml (hunk 1 above)
Comment (dansjack) [reply .idea/misc.xml:3]: This change happened automatically when I merged main into develop

**Review by AllisonBC — DISMISSED** [review]
Approve additional 'compare' code

**Review by AllisonBC — DISMISSED** [review]
Additional changes approved

**Review by AllisonBC — APPROVED** [review]

**Review by unicornsleuth — APPROVED** [review]
Looks great!

**Pull request merged**